admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -273,53 +212,57 @@ Upgrade from Archivematica |previous_version| for CentOS/Red Hat
    you should be able to clear the cache with control-shift-R or
    command-shift-F5.
 
-Update Transfer Index
-^^^^^^^^^^^^^^^^^^^^^
-
-This new feature allows you to update a backlog created in an earlier version
-of Archivematica so that it can be used with the Appraisal Tab and the Backlog
-tab. To test this feature, follow the instructions
-
-.. IMPORTANT::
+.. _upgrade-search-indices:
 
-   These are experimental instructions. Do not use them on a production system
-   unless you have a back-up you can restore from.
+Upgrade search indices
+----------------------
 
-1. Install devtools
+.. note::
 
-   .. code:: bash
+   Ignore this section if you are not planning to use the searching
+   capabilities of Archivematica. Instead, follow the instructions on `how to
```

Comment:
Maybe instead, "Ignore this section if you are planning to run Archivematica without a search index."